app/default/routes.py

```diff
@@ -301,6 +346,14 @@ def internal_server_error(error):
     current_app.logger.error(f"Encountered 500: {error}")
     return render_template("500.html"), 500
 
+@default_bp.errorhandler(401)
+def unauthorised_error(error):
+    current_app.logger.error(f"Encountered 401: {error}")
+    round_data = get_round_data_fail_gracefully(
```

Comment:
We use a mix of get_round_data and get round_data_gracefully, am I right in thinking we use the graceful get when we are due to send the info to the notification store - so the error surfaces there instead?

Reply:
So the graceful one was specifically introduced for the error pages - so if it errors getting round data it just returns and empty object rather than throw an error. Then the error page deals with an empty error. Otherwise we had the situation where trying to display the error page could result in an error...

Reply:
ahh gotcha, okay thanks for clarifying 👍